### PR TITLE
added: contribution guidelines file

### PR DESCRIPTION
**Pull request by suhasasumukh**
To ensure a safe, respectful, and inclusive environment for all contributors and users. This code will
serve as a guideline for behavior, promoting diversity, reducing conflicts, and attracting a
wider range of perspectives.

fixes: https://github.com/OpenBMB/ChatDev/issues/238

### CODE REVIEW

**Comment by cryptictech** [comment]
Actually, this appears to be a duplicate of #239

File: Contribution_Guidelines.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+## Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
```
[the remark this answers is not in the document]
Comment (suhasasumukh) [reply Contribution_Guidelines.md]: its should be fixed with the new update

**Comment by suhasasumukh** [comment]
@cryptictech please look at the latest push - I have outlined the guidelines better

**Comment by suhasasumukh** [comment]
@nenuadrian @cryptictech please look into this commit

**Pull request closed without merging**